### DISCUSSION TITLE
Your CloudWatch Queries & Dashboards

### DISCUSSION THREAD

**Discussion by metaskills** [customink/lambdakiq#3] [Show and tell]
## 👋 Welcome! 

How are you using Lambdakiq's CloudWatch Embedded Metrics? Please share your Metric of Insights snippets.

**Comment by fabiensebban** [comment]
I don't use it a lot. Only to know if a Lambda takes a long time to run.
Is there a way to disable de CloudWatch events? They make CloudWatch service cost a lot (~ $60) compare to Lambda service (~ $0,05) in production.

**Comment by metaskills** [comment]
Yup: https://github.com/customink/lambdakiq#rails-configs you can set the metrics logger. For example:

```ruby
config.lambdakiq.metrics_logger = Logger.new("/dev/null")
```

**Reply by fabiensebban** [reply]
Hi @metaskills!

I'm so sorry to bring this topic back after so long. I tried to add the following:

In `production.rb` file
```ruby
  logger = Logger.new("/dev/null")
  config.logger    = logger

  config.log_level = :info
  config.lograge.enabled = true
  config.lograge.formatter = Lograge::Formatters::Json.new
  config.lograge.custom_payload do |controller|
    { requestid: controller.request.request_id }
  end

  config.lambdakiq.metrics_logger = Logger.new("/dev/null")
```

In `app.rb` file:
```ruby
Lambdakiq.config.metrics_logger = Logger.new("/dev/null")
```

CloudWatch metrics are still sent to AWS.

Do you know how to stop CloudWatch metrics?
